### PR TITLE
add file type aac

### PR DESCRIPTION
**Pull request by duybq86**
add file type aac

### CODE REVIEW

File: resources/config/mimetypemapping.dist.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 	"3gp": ["video/3gpp"],
 	"7z": ["application/x-7z-compressed"],
+	"aac": [""audio/aac""],
```
Comment (danielkesselberg) [inline resources/config/mimetypemapping.dist.json]: There are to many double quotes. 

`"aac": ["audio/aac"],`

**Review by kesselb — CHANGES_REQUESTED** [review]
Thank you for your contribution :tada:

**Comment by ChristophWurst** [comment]
Any update @duybq86? Did you delete your fork, btw?

**Comment by MorrisJobke** [comment]
Due to deleted fork I will close this PR for now. We are happy to help once this is needed again.

**Pull request closed without merging**